app/models/sample.rb

```diff
@@ -179,11 +179,24 @@ def initiate_input_file_upload
   def initiate_s3_cp
     return unless status == STATUS_CREATED
     stderr_array = []
+    total_reads_json = "total_reads.json"
+    total_reads_json_path = nil
     input_files.each do |input_file|
       fastq = input_file.source
+      total_reads_json_path = File.join(File.dirname(fastq.to_s), total_reads_json)
       _stdout, stderr, status = Open3.capture3("aws", "s3", "cp", fastq.to_s, "#{sample_input_s3_path}/#{input_file.name}")
       stderr_array << stderr unless status.exitstatus.zero?
     end
+    if total_reads_json.present?
```

Comment:
did you mean `if total_reads_json_path.present?` ?

Reply:
Yes!

Reply:
Thanks for catching this :)